test/components/plots/stackedPlotTests.ts

```diff
@@ -267,6 +267,61 @@ describe("Plots", () => {
     });
   });
 
+  describe("fail safe tests", () => {
+
+    var svg: D3.Selection;
+    var xScale: Plottable.Scale.Category;
+    var yScale: Plottable.Scale.Linear;
+    var data1: Object[];
+    var data2: Object[];
+
+    beforeEach(() => {
+      svg = generateSVG(600, 400);
+
+      data1 = [
+          { x: "A", y: "s", fill: "blue" },
+      ];
+      data2 = [
+          { x: "A", y: 1, fill: "red"},
+      ];
+      xScale = new Plottable.Scale.Category();
+      yScale = new Plottable.Scale.Linear();
+    });
+
+    afterEach(() => {
+      svg.remove();
+    });
+
+    it("conversion fails should be silent in stackedBarPlot", () => {
+      var plot = new Plottable.Plot.StackedBar(xScale, yScale);
+      plot.addDataset("d1", data1);
+      plot.addDataset("d2", data2);
+      plot.project("fill", "fill");
+      plot.project("x", "x", xScale).project("y", "y", yScale);
+
+      var ds1PlotMetadata = <Plottable.Plot.StackedPlotMetadata>(<any> plot)._key2PlotDatasetKey.get("d1").plotMetadata;
+      var ds2PlotMetadata = <Plottable.Plot.StackedPlotMetadata>(<any> plot)._key2PlotDatasetKey.get("d2").plotMetadata;
+
+      assert.isFalse(isNaN(ds1PlotMetadata.offsets.get("A")), "ds1 offset should be a number");
+      assert.isFalse(isNaN(ds2PlotMetadata.offsets.get("A")), "ds2 offset should be a number");
```

Comment:
`"string" !== "another string"` also returns `true`. How about checking for undefined then? or checking `typeof x === "number"`